src/script/interpreter.cpp

```diff
@@ -83,12 +83,18 @@ static void CleanupScriptCode(CScript &scriptCode,
 static bool IsOpcodeDisabled(opcodetype opcode, uint32_t flags) {
     switch (opcode) {
         case OP_RESERVED:
+        case OP_VER:
```

Comment:
Are these things supposed to be in this commit?

Reply:
No. Will fix.